src/main/java/com/schematica/world/schematic/UnsupportedFormatException.java

```diff
@@ -2,6 +2,7 @@
 
 class UnsupportedFormatException extends Exception
 {
+    static final long serialVersionUID = -987325498762354L;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Explicitly declare the visibility for "serialVersionUID". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2039)